Before: aae6beccd610ed4a77848a05be6444e0a5480480be75fa8c12c77478c7ac9dc6
After: abe19e0cefa22500e8c35336a6fcfdb66edaed187adfb94e746d438f0ea18f37
Patch: 13e868ccfc994490e6bd718b404a48763628d07f515799d0a3c96af13e172f1c
--- lib/%project_name%/%feature_name%_feature.rb.tt	2012-09-06 08:00:49.576741943 +1000
+++ lib/%project_name%/%feature_name%_feature.rb.tt	2012-09-06 08:00:54.596771116 +1000
@@ -5,12 +5,7 @@
 
     since_ver <%= since_ver %>
 
-    corvid_tasks {
-      # require '<%= project_name %>/example/corvid_task'
-    }
-
     rake_tasks {
-      # require '<%= project_name %>/example/task.rake'
     }
 
   end
